src/Stratis.Features.Collateral/CollateralPoAMiner.cs

```diff
@@ -121,31 +120,32 @@ private void OnBeforeFillBlockTemplate()
 
             try
             {
-                List<Poll> pendingAddFederationMemberPolls = this.votingManager.GetPendingPolls().Where(p => p.VotingData.Key == VoteKey.AddFederationMember).ToList();
+                List<Poll> pendingMemberPolls = this.votingManager.GetPendingPolls().MemberPolls().ToList();
```

Comment:
Both adding and kicking of members should check that the conditions are met to do so. We can't simply vote because some other node did. Since the conditions for kicking is checked on an on-going basis (elsewhere) I'm not sure we need to include it here.